.github/workflows/percy.yml

```
@@ -6,21 +6,20 @@ on:
       - master
       - main
   pull_request_target:
+    types: [opened, reopened, synchronized, labeled]
```

actionlint (1.7.4)

invalid activity type "synchronized" for "pull_request_target" Webhook event. available types are "assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "edited", "labeled", "locked", "opened", "ready_for_review", "reopened", "review_request_removed", "review_requested", "synchronize", "unassigned", "unlabeled", "unlocked"

(events)